docs/configuration.rst

```diff
@@ -139,7 +139,16 @@ multiple choices of matching are possible.
 We could match ``seg1/seg2`` with the first token, and ``seg3`` with the second token,
 we could also match ``seg1`` with the first token, and ``seg2/seg3`` with the second token, etc.
 
-The variables and tokens are useful to know the type of any segments that doesn't have a fixed name.
+The :key:`field` : This is useful when we want to have a different mapping between resources. The resource_name
```

Comment:
I think `resource_name` should be in `` characters, like the `nametype_path` below, since it is the name of a key.

Also, just not sure about the `:key:` if it works. Does it render correctly if you generate the doc? Maybe just verify this, and adjust the text and/or syntax if needed to make it look correctly when the doc is rendered. 

After this, we should be good to go!

Reply:
When I use the `make docs` command to render the rst as html files and I open the page on my browser, it doesn't seem to work for me. 

![image](https://github.com/Ouranosinc/cowbird/assets/36516122/bc44e586-a305-47d2-a020-0c9a7ed983be)

The `:data:` comes from the [python-specific directive](https://www.sphinx-doc.org/en/master/usage/domains/python.html#directive-py-data) from sphinx and creates a link to the same-name variable in the code, but I'm not sure `key` exists with sphinx.
I think just using a double `` for these key names would be okay here.

Reply:
Okay thanks